packages/flutter_goldens/test/comparator_selection_test.dart

```diff
@@ -60,18 +60,18 @@ void main() {
     expect(_testRecommendations(hasFlutterRoot: true, hasGold: true), _Comparator.local);
 
     // If we don't have gold but are on CI, we skip regardless.
-    expect(_testRecommendations(hasLuci: true), _Comparator.local); // TODO(ianh): this should be skip
```

Comment:
Was this a TODO instead of immediately fixing it because of worries it would be a breaking change?

Reply:
Yeah, exactly.

After we land this, it might be worth checking to see if everything is working as expected, as this change could I guess be part of the issue that led to the regression of #139549 (I mean, I can't see why it would, but then I can't see why _anything_ in that PR would, so...).